R/bibstats.R

```diff
@@ -76,10 +81,17 @@ percent_female <- function(bib_file,
                                                  ".*,[:blank:]")
       }
     }
-    gender <- sapply(authors,
-                     function(x) any(gender::gender(x, method = "genderize")$gender == "female"))
-    paste0(round(sum(gender) / length(gender), 2) * 100,
-                 "% female authors")
+    gender <- tryCatch({
+      sapply(authors, function(x)
+        any(gender::gender(x, method = "genderize")$gender == "female"))
+      },
+      message = "Rate limit for the `{gender}` R package has been reached due to the amount of download requests.
+      Please wait a few hours and try again later."
```

Comment:
Do we need to force a slow down in requests somehow so that the package is not being 'antisocial'?